ereol_app_feeds/src/Controller/DefaultController.php

```diff
@@ -54,6 +54,22 @@ public function paragraphs($type) {
     drupal_json_output($data);
   }
 
+  /**
+   * Render Overdrive mappings.
+   */
+  public function overdriveMapping() {
+    $triggers = overdrive_triggers_load();
+
+    $data = array_map(function ($trigger) {
+      return [
+        'trigger' => $trigger['search_trigger'],
+        'query' => $trigger['search_query'],
+      ];
+    }, $triggers);
+
+    drupal_json_output($data);
+  }
```

Comment:
Maybe an `drupal_exit()` for faster shutdown.

Reply:
This will stop https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_page_footer/7.x from getting called.

Reply:
I don't understand this comment. Will `drupal_exit()` stop anything from getting called, or should I add `drupal_exit()`?